src/Elastic.Apm.StartupHook.Loader/Loader.cs

```diff
@@ -57,15 +57,24 @@ private static void StartAgent()
 			Agent.Setup(new AgentComponents());
 			Agent.Subscribe(new HttpDiagnosticsSubscriber());

```

Comment:
Should also be done inside `LoadDiagnosticSubscriber(new HttpDiagnosticsSubscriber())`?

Reply:
Yeah, it definitely doesn't hurt. 

`HttpDiagnosticsSubscriber` lives within `Elastic.Apm` and only depends on things that are always loaded, so I don't expect that to fail. Nevertheless I agree it's best to also guard this one. Moved it into `LoadDiagnosticSubscriber()`.